src/eterna/ui/Toolbar.ts

```diff
@@ -294,210 +440,186 @@ export default class Toolbar extends ContainerObject {
             .hotkey(KeyCode.KeyJ);
 
-        this.actionMenu.addSubMenuButton(0, this.nucleotideFindButton);
+        this.addObject(this.nucleotideFindButton, this.middle);
+        this.availableButtons.push(this.nucleotideFindButton);
 
-        this.nucleotideRangeButton = new GameButton()
+        this.nucleotideRangeButton = new ToolbarButton()
             .allStates(Bitmaps.NovaPuzzleImg)
-            .disabled()
-            .label('View Nucleotide Range', 14)
-            .scaleBitmapToLabel()
             .tooltip('Enter a nucleotide range to view (v)')
             .hotkey(KeyCode.KeyV);
 
-        this.actionMenu.addSubMenuButton(0, this.nucleotideRangeButton);
+        this.availableButtons.push(this.nucleotideRangeButton);
 
-        this.explosionFactorButton = new GameButton()
+        this.explosionFactorButton = new ToolbarButton()
             .allStates(Bitmaps.ImgFlare)
             .disabled()
-            .label('Explosion Factor', 14)
-            .scaleBitmapToLabel()
             .tooltip('Set explosion factor ([, ])');
 
-        this.actionMenu.addSubMenuButton(0, this.explosionFactorButton);
+        // this.addObject(this.explosionFactorButton, this.middle);
+        this.availableButtons.push(this.explosionFactorButton);
 
         this.boostersMenu = new GameButton().allStates(Bitmaps.NovaBoosters).disabled(undefined);
 
-        if (this._boostersData != null && this._boostersData.actions != null && this._showAdvancedMenus) {
-            const boosterMenuIdx = this.actionMenu.addMenuButton(this.boostersMenu);
-            for (let ii = 0; ii < this._boostersData.actions.length; ii++) {
-                const data = this._boostersData.actions[ii];
-                Booster.create(this.mode as PoseEditMode, data).then((booster) => {
-                    const button: GameButton = booster.createButton(14);
-                    this.regs.add(button.clicked.connect(() => booster.onRun()));
-                    this.actionMenu.addSubMenuButtonAt(boosterMenuIdx, button, ii);
-                    this.dynActionTools.push(button);
-                });
-            }
-        }
-
-        this.moveButton = new GameButton()
+        // if (this._boostersData != null && this._boostersData.actions != null && this._showAdvancedMenus) {
+        //     const boosterMenuIdx = this.actionMenu.addMenuButton(this.boostersMenu);
+        //     for (let ii = 0; ii < this._boostersData.actions.length; ii++) {
+        //         const data = this._boostersData.actions[ii];
+        //         Booster.create(this.mode as PoseEditMode, data).then((booster) => {
+        //             const button: GameButton = booster.createButton(14);
+        //             this.regs.add(button.clicked.connect(() => booster.onRun()));
+        //             this.actionMenu.addSubMenuButtonAt(boosterMenuIdx, button, ii);
+        //             this.dynActionTools.push(button);
+        //         });
+        //     }
+        // }
```

Comment:
We need this for registering boosters - likely we'll need a new button that brings up a list of boosters that can be run